client/src/components/Common/VegaWrapper.vue

```diff
@@ -1,11 +1,16 @@
 <template>
-    <div ref="chartContainer" :style="style"></div>
+    <div>
```

Comment:
`Common`, `util` and `helpers` is probably the worst antipattern ? Call it `PlottingWrappers` or something like that ?

Reply:
But ... these are not visualizations, those mean something else in Galaxy ?